content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,149 @@
+
+The party planner (oracle) must have:
+
+- The ability to generate a condition and a fulfillment.
+
+- Access to a [`rippled` server](install-rippled.html) that they can use to submit the fulfillment value and finish the escrow.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## Define the terms of the smart contract
+
+To create the escrow as a smart contract, the participants must first define the terms of the contract. In this scenario, the participants need to agree on the following details.
+
+- Should the escrow be fulfilled after a specific time has passed or after a cryptographic condition has been fulfilled?
+
+    - *[Conditionally-held escrow](send-a-conditionally-held-escrow.html)*: In this case, the participants agree to use a conditionally-held escrow. This is a good choice because the smart contract can be fulfilled only after the party planner (oracle) verifies that the party band completed their set and submits the cryptographic condition and fulfillment to finish the escrow and release the XRP.
+
+    To handle a no-show scenario, the participants configure the escrow to be eligible for cancellation after a specific time. Typically, this cancellation time should be after the time at which the party planner is expected to finish the escrow. The the cancellation time in this way helps prevent the party host from maliciously cancelling the escrow before the party planner can finish the escrow.
+
+    The conditionally-held escrow provides the functionality required to handle the terms of this smart contract by putting the ability to finish the escrow solely in the hands of the neutral party planner (oracle).
+
+    - *[Time-held escrow](send-a-time-held-escrow.html)*: A time-held escrow is not the best choice for this scenario because the party band could be a no-show. If the party band is scheduled to finish their set at 11 p.m., the participants could agree to allow the escrow to finish and release the XRP at 11 p.m. At 11 p.m., any participant, including the party band, can finish the escrow and release the XRP, even if the band didn't show up. This does not meet the terms of the smart contract.
+
+    The participants could agree to allow the cancellation of the escrow before 11 p.m. to accommodate the no-show scenario. However, consider that the party band could show up and play their set, but the party host cancels the escrow before a participant can finish the escrow at 11 pm.
+
+- Should the escrow disallow fulfillment until a specific time?
+
+  The participants agree to disallow the escrow from being fulfilled until 11 p.m. the night of the party, which is when the party band is scheduled to complete their set. ***TODO: not necessary, but just an extra precaution? Or just unnecessary?***
+
+- Should the participants be able to cancel the escrow after a specific time period if the condition is not fulfilled?
```

Comment:
> Should the participants be able to cancel the escrow

Suggest changing this to "Should the escow expire" and the answer is "absolutely yes" because otherwise the only outcomes are:

- The party planner releases the condition: the band gets paid
- The party planner doesn't release the condition: the money stays locked in escrow forever